package.json

```diff
@@ -45,6 +45,7 @@
     "url": "https://github.com/nestjs-architects/typed-cqrs/issues"
   },
   "devDependencies": {
+    "@nestjs-architects/typed-cqrs": "file:./",
```

Comment:
Why we need it?

Reply:
We don't, I can remove it from the PR if you want, but this removes the need to call `npm link` in order for the examples to typecheck correctly.

Reply:
Removed and rebased